src/flow_id.rs

```diff
@@ -206,4 +217,10 @@ mod tests {
         flow2.hash(&mut hasher2);
         assert_eq!(hasher1.finish(), hasher2.finish());
     }
+
+    #[test]
+    fn test_display() {
+        let flow = build_local_flow_id();
+        assert_eq!(format!("{}", flow), "127.0.0.1-192.168.0.1-8001-8002-17")
```

Comment:
Yes.
We may use into our code the method to_string() dedicated to variable (format is more log oriented:

        assert_eq!(flow.to_string(), "127.0.0.1-192.168.0.1-8001-8002-17");

Reply:
to trigger the deployments (Crates and Docs), you must increase the version into Cargo.toml:

[package]
name = "libflow"
version = "0.1.4"

Reply:
👍